src/modules/git_commit.rs

```diff
@@ -335,4 +339,76 @@ mod tests {
         assert_eq!(expected, actual);
         Ok(())
     }
+
+    #[test]
+    fn test_latest_tag_shown_with_tag_enabled() -> io::Result<()> {
+        use std::{thread, time};
+
+        let repo_dir = fixture_repo(FixtureProvider::GIT)?;
+
+        let mut git_commit = Command::new("git")
+            .args(&["rev-parse", "HEAD"])
+            .current_dir(&repo_dir.path())
+            .output()?
+            .stdout;
+        git_commit.truncate(7);
+        let commit_output = str::from_utf8(&git_commit).unwrap().trim();
+
+        Command::new("git")
+            .args(&["tag", "v2", "-m", "Testing tags v2"])
+            .current_dir(&repo_dir.path())
+            .output()?;
+
+        // Wait one second between tags
+        thread::sleep(time::Duration::from_millis(1000));
+
+        Command::new("git")
+            .args(&["tag", "v0", "-m", "Testing tags v0", "HEAD~1"])
+            .current_dir(&repo_dir.path())
+            .output()?;
+
+        Command::new("git")
+            .args(&["tag", "v1", "-m", "Testing tags v1"])
+            .current_dir(&repo_dir.path())
+            .output()?;
+
+        let git_tag = Command::new("git")
```

Comment:
Just to confirm, the output format of this shouldn't change with newer git versions right? Git generally doesn't consider the output text of commands to be an API contract, but it looks to me like there's enough specifiers in here that there's not really a way git could change the output without rendering the flags broken?

Reply:
Yes, this output shouldn't change with newer git releases. If so, I think we should find another way to get the latest tag on a commit...

Reply:
Okay, sounds good!